cmd/amppkg/main.go

```diff
@@ -126,8 +127,9 @@ func main() {
 		}
 	}
 
+	signerRequireHeaders := !*flagDevelopment
 	signer, err := signer.New(certCache, key, config.URLSet, rtvCache, certCache.IsHealthy,
-		overrideBaseURL, /*requireHeaders=*/!*flagDevelopment, config.ForwardedRequestHeaders)
+		overrideBaseURL, signerRequireHeaders, config.ForwardedRequestHeaders)
```

Comment:
gofmt was breaking formatting here.

Reply:
What did gofmt break?

Reply:
It breaks the inline comment (/*requireHeaders=*/)

Before gofmt:

```
	signer, err := signer.New(certCache, key, config.URLSet, rtvCache, certCache.IsHealthy,
		overrideBaseURL, /*requireHeaders=*/!*flagDevelopment, config.ForwardedRequestHeaders)
```

After:

```
	signer, err := signer.New(certCache, key, config.URLSet, rtvCache, certCache.IsHealthy,
		overrideBaseURL /*requireHeaders=*/, !*flagDevelopment, config.ForwardedRequestHeaders)
```

Reply:
Might be a bug with gofmt.  Although https://golang.org/doc/effective_go.html#commentary says that /* */ is mostly reserved for package comments and disabling code. I vote for making gofmt happy.

Reply:
SG!